### ISSUE TITLE
Error Code = 0x80070050

### ISSUE THREAD

**Issue by confused-Techie** [felixrieseberg/electron-windows-store#140]
Attempting to run the Electron-Windows-Store tool seems to continually error out. There seems to be two issues that may have the same root cause. 

After a few successful runs of the application a few days ago with no major system changes, running the exact same command (even used my bash history) fails.

Command:
```electron-windows-store --input-directory "D:\App Location\App" --output-directory D:\Output --package-name AppName --package-version 1.0.0.0 --publisher-display-name 'App Name' -a D:\Resources\```

Output:
```
Creating Certificate
Succeeded

ERROR: Output file exists.
(Error Code = 0x80070050).

Error: C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64\pvk2pfx.exe exited with code: 1
    at ChildProcess.<anonymous> (C:\Users\antho\AppData\Roaming\npm\node_modules\electron-windows-store\lib\utils.js:74:23)
    at ChildProcess.emit (events.js:315:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
Error: C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64\pvk2pfx.exe exited with code: 1
    at ChildProcess.<anonymous> (C:\Users\antho\AppData\Roaming\npm\node_modules\electron-windows-store\lib\utils.js:74:23)
    at ChildProcess.emit (events.js:315:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
```

As well as every time running the app it asks the one-time setup questions. Even though the .electron-windows-store file is created successfully, with:

```
{
  "desktopConverter": false,
  "expandedBaseImage": false,
  "publisher": "developmentca",
  "windowsKit": "C:\\Program Files (x86)\\Windows Kits\\10\\bin\\10.0.19041.0\\x64"
}
```

I have used the built in tools in windows to repair the install of 'Windows Software Development Kit - Windows 10.0.19041.685' and reinstalled electron-windows-store using
```
npm uninstall -g electron-windows-store
npm install -g electron-windows-store
```

But the command is still failing. Any ideas?

**Comment by confused-Techie** [comment]
TLDR. Delete any certificates which may exist in ```C:\Users\[User Name]\AppData\Roaming\electron-windows-store\[Publisher ID]\```

Taking a look at the source I was able to see where the command was failing, after ```lib/setup.js``` Line 175 calls ```sign.makeCert``` Line 97 of ```lib/sign.js``` calls the process pvk2pfx.exe in the Windows SDK, which is provided with the arguments of where to install the certificate based off of ```process.env.APPDATA, 'electron-windows-store', publisher```. Meaning that if the file already exists pvk2pfx.exe will fail when attempting to run, since electron-windows-store never checks if the publisher ID Certification file may already exist, which would be a valid way to fix this error.
